suites/nautilus/rgw/sanity_rgw_multisite_2.yaml

```diff
@@ -0,0 +1,372 @@
+tests:
```

Comment:
IMO, adding documentation about the objective of the test module helps in the long run. It also provides an overview to other team members.

Reply:
Done.